topics/contributing/tutorials/fair-by-design/tutorial.md

```diff
@@ -0,0 +1,1065 @@
+>    - If so, do you regularly gather and analyse that feedback?
+> - Does the resource adopt an open community approach regarding its quality and reachability?
+> - Has the learning resource been checked by a third party regarding its learning experience quality?
+{: .details}
+
+### Feedback QA
+
+> <comment-title>Regularly gather feedback from learners and instructors</comment-title>
+> Ensure that you actively and regularly gather feedback from both perspectives: the learners and the instructors.
+{: .comment}
+
+<a href="https://fair-by-design-methodology.github.io/FAIR-by-Design_Book/4%20-%20FAIR-by-design%20learning%20materials%20creation/4.1%20-%20Workflow%20stages%20description/416-verify/" class="btn btn-dark text-white btn-lg btn-block">Go to the full FAIR-by-Design Methodology: Verify stage ...</a>
+
+## Stage 7 - Continuous Improvement
+> To improve is to change; to be perfect is to change often
+{: .quote author="Winston Churchill"}
+
+#### {% icon galaxy-download %} Gather
+
+- Gather feedback from all available internal & external sources.
+- Potential sources:
+	- Feedback form
+	- QA recommendations
+	- Self-reflection after training
+	- Git Issues
+	- Gitter Chat
+	- Direct mail contact
+	- Other means of communication
+
+#### {% icon galaxy-barchart %} Analyse
+
+- Analyse the gathered information in a structured way.
+- Create a list of potential improvements with impact level (high, moderate, low).
+
+#### {% icon galaxy-wf-edit %} Improve
+
+- Select items from the list that will be part of a new version.
+- Choose items that make sense to be in the same new release.
+
+#### {% icon galaxy-history-refresh %} Repeat
+
+- Start a new cycle of the FAIR-by-Design methodology that will implement the selected items.
+- After the Verify stage, you will reenter continuous improvement with the newly gathered information.
+
+<a href="https://fair-by-design-methodology.github.io/FAIR-by-Design_Book/4%20-%20FAIR-by-design%20learning%20materials%20creation/4.2%20-%20Continuous%20Improvement/417-improvement/" class="btn btn-dark text-white btn-lg btn-block">Go to the full FAIR-by-Design Methodology: Continuous Improvement ...</a>
+
+# About
+
+## FAIR-by-Design Microlearning
+
+### Self-paced how to FAIR-by-Design guide
+
+### Original Location
+
+- online: [https://fair-by-design-methodology.github.io/microlearning/latest/](https://fair-by-design-methodology.github.io/microlearning/latest/)
+- this tutorial is an adapted version of the original microlearning guide
+
+### Training Description
+
+Short how to guide that guides you through the stages of the FAIR-by-Design methodology without any specific choice on tools and formats.
+
+### Target audience
+
+All interested parties who need to develop learning materials for any type of project-related training.
+
+### Expertise Level / Skill Level: Beginner
+
+### Primary Language: English
+
+### Access Cost: No
```

Comment:
A few of those are already in the metadata at the top, is it necessary to repeat them here?

Reply:
Yes, please keep them here also. We need to provide the full set of metadata according to the RDA Schema and Skills4EOSC internal rules.

Reply:
@bebatut I think that is a good idea. Fields like audience we should definitely add to the GTN supported metadata. Then we can list any metadata that is not yet mentioned elsewhere in the tutorial here at the bottom, to ensure it also adheres to the skills4EOSC guidelines